src/page/AddMenu/components/MenuDetail/MenuDetail.module.scss

```
@@ -0,0 +1,70 @@
+.container {
+  margin-top: 32px;
+}
+
+.caption {
+  display: flex;
+  align-items: center;
+  height: 36px;
+  color: #175c8e;
+  font-size: 24px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
+}
+
+.menu-detail-input {
```

`input`같은 경우에는 BEM방식을 적용하는게 나을거 같아요..!
```suggestion
.menu-detail__input {
```

---

메뉴 상세정보를 넣는 Input여서 detail-input이라고 작성하였습니다!